### PR TITLE
update SIP committee membership list

### PR DESCRIPTION
**Pull request by SethTisue**
also tweeted: https://twitter.com/scala_lang/status/1091089089107877889  fyi @smarter

### CODE REVIEW

File: _sips/sip-submission.md [hunk 1]
```diff
@@ -197,16 +197,17 @@ The current committee members are:
...
+- Sébastien Doeraene ([@sjrd](https://github.com/sjrd)), Scala Center
 - Eugene Burmako ([@xeno-by](https://github.com/xeno-by)), Twitter
 - Josh Suereth ([@jsuereth](https://github.com/jsuereth)), Google
 - Iulian Dragos ([@dragos](https://github.com/dragos)), Triplequote
+- Guillaume Martres ([@smarter](https://github.com/smarter)), EPFL
 
 The current Process Lead is:
 
-- Jorge Vicente Cantero ([@jvican](https://github.com/jvican)), Scala Center
+- Darja Ivanovic ([@darjutak](https://github.com/darjutak)), Scala Center
```
Comment (sjrd) [inline _sips/sip-submission.md]: It's Jovanovic.

Comment (SethTisue) [reply _sips/sip-submission.md]: Gah

Comment (SethTisue) [reply _sips/sip-submission.md]: gah, where'd I get "Ivanovic"? fixed

**Comment by SethTisue** [comment]
@sjrd tell me I didn't get Guillaume's name wrong in the tweet :-)

**Comment by SethTisue** [comment]
the CI failure is unrelated link checker stuff

**Pull request merged**